packages/wdio-spec-reporter/tests/index.test.ts

```diff
@@ -395,10 +396,9 @@ describe('SpecReporter', () => {
             expect(result).toMatchSnapshot()
         })
 
-        it('should not print if data table format is not given', () => {
+        it('should print doc string', () => {
```

Comment:
Why do we replace the test? Can we just add one?

Reply:
Currently, this case tests if the `argument` is a string, there will nothing to show. The new behavior is when the `argument` is a string, the doc string will be printed in the report for the related test. Behavior changes required this case to be replaced IMO.

Reply:
We also want to ensure nothing bad happens if the argument is empty, so let's keep the existing test and just add a new one.

Reply:
Please check my solution for this concerning test case.